### ISSUE TITLE
Link Error with CMake

### ISSUE THREAD

**Issue by carbotaniuman** [twistedfall/opencv-rust#137]
Before reporting an issue please first check the
[troubleshooting guide](https://github.com/twistedfall/opencv-rust/blob/master/README.md#troubleshooting). If
the issue you're encountering is not solved thereby please state the following in your bugreport:
1. Windows 10 2004
2. vcpkg - also installed llvm per readme.
3. opencv 4.1.1
4. Attach the full output of the following command from your project directory:

Dependency block as so:
```
opencv = {git = "https://github.com/twistedfall/opencv-rust/", features = ["buildtime-bindgen"]}
```

Abridged: https://gist.github.com/carbotaniuman/c47a6e689610cb03b61fca118ea321ca
Raw: https://gist.github.com/carbotaniuman/c8f66d1b1e5fb3bcae2e1f18b88b1f46

With `$env:OPENCV_DISABLE_PROBES="cmake"` and `$env:VCPKGRS_DYNAMIC=1`, the code compiles and works.

**Comment by twistedfall** [comment]
I see, I’ll bump the priority of vcpkg, it should come before cmake at least on Windows.

**Comment by twistedfall** [comment]
Will you please check the current master to see if it fixes the problem for you? You should no longer need the `OPENCV_DISABLE_PROBES` env variable, but `VCPKGRS_DYNAMIC` might still be needed (out of my control).

**Comment by carbotaniuman** [comment]
That appears to have fixed it. Thanks!

**Comment by twistedfall** [comment]
Fixed in 0.40.1
